frontend/app/src/components/accounts/AccountBalances.vue

```diff
@@ -181,6 +181,8 @@ const deleteAccount = async () => {
     set(xpubToDelete, null);
     await deleteXpub(payload);
     get(balanceTable)?.removeCollapsed(payload);
+
+    await refreshAccounts(get(blockchain));
```

Comment:
any chance we can make `refreshAccounts` receive a `MaybeRef` and handle the get inside?